Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/EditorMeshComponent.cpp

```diff
@@ -112,6 +131,14 @@ namespace AZ
                     ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation);
 
                 behaviorContext->Class<EditorMeshComponent>()->RequestBus("RenderMeshComponentRequestBus");
+
+                behaviorContext->EBus<MeshComponentNotificationBus>("MeshComponentNotificationBus")
+                    ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation)
+                    ->Attribute(AZ::Script::Attributes::Category, "Editor")
+                    ->Attribute(AZ::Script::Attributes::Module, "editor")
+                    ->Handler<Internal::MeshComponentNotificationBusHandler>()
+                    ->Event("OnModelReady", &MeshComponentNotifications::OnModelReady)
+                    ;
```

Comment:
This should be added to the behavior context in the controller instead, with the MeshComponentRequestBus.
It should use the common scope so that it can also be used in lua and py.
The category and module should match whenever is used for MeshComponentRequestBus.  
We will need to do a pass later for all of the notification buses that have not been registered.

Reply:
Done.